trinity/utils/hexadecimal.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+
+import codecs
+
+
+def encode_hex(value):
+    return '0x' + codecs.decode(codecs.encode(value, 'hex'), 'utf8')
+
+
+def decode_hex(value):
+    _, _, hex_part = value.rpartition('x')
+    return codecs.decode(hex_part, 'hex')
```

Comment:
Why can't we use the eth-utils ones here (and in py-evm)? Seems like a lot of test and implementation duplication.

Reply:
I've been considering the `eth_utils` ones off-limits until we convert API with the new non-`force_text` based versions.  Also, because of the extra handling for both text and binary types they have extra overhead which was why I kept them out of py-evm (may not be necessary anymore, decision from a long time ago).

I don't love all the duplicate utils, but hate it less than I hate using the `eth_utils` versions prior to converting them....  dunno, I could go either way.  I'll sleep on it and if you feel strongly I'll take your lead.